compiler/circle-eval-diff/include/CircleEvalDiff.h

```diff
@@ -33,6 +33,7 @@ enum class Metric
 {
   Undefined, // For debugging
   MAE,
+  MAPE,
```

Comment:
(optioanl) maybe add `// Mean Absolute Percentage Error` ?

Reply:
I'll add comments for both MAE and MAPE in another PR.